aquifer/aquifer
# Automatically generate settings files

Generates settings files automatically during the build.

To test:
Create and build as normal.

Ensure local.settings.php and secret.settings.php are generated.
